tests/integration/medusa_s3_integration_test.go

```diff
@@ -0,0 +1,60 @@
+package integration
+
+import (
+	. "github.com/k8ssandra/k8ssandra/tests/integration/steps"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var (
+	medusaS3TestTable    = "medusa_test"
+	medusaS3TestKeyspace = "medusa"
+)
+
+var _ = Describe("Medusa on S3", func() {
+	Context("Deploy K8ssandra with Medusa", func() {
+		It("Create a kind cluster and a namespace", func() {
```

Comment:
Same code as above test, lets not repeat these steps. This should be ``createCluster()``

Reply:
Sounds good 👍